services/api-server/src/simcore_service_api_server/exceptions/backend_errors.py

```diff
@@ -105,3 +105,18 @@ class PricingPlanNotFoundError(BaseBackEndError):
 class ProjectAlreadyStartedError(BaseBackEndError):
     msg_template = "Project already started"
     status_code = status.HTTP_200_OK
+
+
+class InsufficientNumberOfSeatsError(BaseBackEndError):
+    msg_template = "Not enough available seats. Current available seats {num_of_seats} for license item {licensed_item_id}"
+    status_code = status.HTTP_409_CONFLICT
```

Comment:
Q: the status code is inherent to a REST interface. I am just wondering whether it makes sense to have these for internal errors. Just a thought, and actually in the api-server I guess this makes sense. so no worries.